ci-build.sh

```diff
@@ -118,8 +118,8 @@ start_test "Start with minimal settings" "${STD_CMD} \
            -e \"PROXY_SERVICE_PORT=80\""
 
 echo "Test it's up and working..."
-wget -O /dev/null --quiet --no-check-certificate https://${DOCKER_HOST_NAME}:${PORT}/
-echo "Test limited protcol and SSL cipher... "
+curl -O /dev/null -sk https://${DOCKER_HOST_NAME}:${PORT}/
```

Comment:
Do you mean `-o` rather than `-O`?

I think we want the `-S` (show error message) and `-f` (fail on server error) flags as well.

Reply:
I think you're right on all counts!